backend/danswer/configs/model_configs.py

```diff
@@ -128,3 +128,6 @@
         logger.error(
             "Failed to parse LITELLM_PASS_THROUGH_HEADERS, must be a valid JSON object"
         )
+
+# Optinal maximum workers (if 0, disregard)
```

Comment:
I don't think we need this, let's revert this. The other comment I would have here is, MAX_WORKERS is not very descriptive. They may get it confused with workers for indexing or workers for background async jobs.

Reply:
Sounds good- can always add if ends up being a real issue (perhaps for rate limiting reasons with provider apis?)

Reply:
Ya, that could be valid, we can add it as needed